docs/_docs/contributing/workflow.md

```diff
@@ -115,6 +115,7 @@ The basics of working with Dotty codebase are documented [here](https://dotty.ep
 | `testOnly dotty.tools.dotc.CompilationTests -- *pos` | Run test (method) `pos` from `CompilationTests` suite.                                                                        |
 | `testCompilation sample`                             | In all test suites, run test files containing the word `sample` in their title.                                               |
 | `scala3-compiler/Test/runMain dotty.tools.printTypes`|  Print types underlying representation                                                                                        |
+| `scaladoc/generateScalaDocumentation`               | Build the documentation website                                                                                              |
```

Comment:
Can we specify which documentation site this means? It might not be obvious for someone if this is the stable lib, reference/api/nightly one, or what exactly "documentation website" means here.

Reply:
Good point, I’ve pushed a fix-up commit to clarify that.